Cargo.toml

```diff
@@ -10,7 +10,7 @@ repository.workspace = true
 license.workspace = true
 
 [features]
-default = ["threading", "stdlib", "zlib", "importlib"]
+default = ["threading", "stdlib", "zlib", "importlib", "ssl"]
```

Comment:
We removed it due to difficult ssl setup issue on windows. is it now resolved?

Reply:
Oh, oops, that was accidentally left behind from testing.